js/src/pages/settings/track-conversions.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+import { useState, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useAppDispatch } from '~/data';
+import useEnableEnhancedConversions from '~/hooks/useEnableEnhancedConversions';
+import useDispatchCoreNotices from '~/hooks/useDispatchCoreNotices';
+import AccountCard from '~/components/account-card';
+import SpinnerCard from '~/components/spinner-card';
+
+const TrackConversions = () => {
```

Comment:
This settings component will also need to be renamed to something related to the Content API (CAPI). Otherwise this component looks really solid! Nice work.

Reply:
Resolving conversation as addressed in (bfcfcb1c0cc57409a421c10f5abc06318740d077))